common/mariadb/templates/initdb/_init.sql.tpl

```diff
@@ -32,6 +32,15 @@ GRANT {{ . }} TO {{ include "mariadb.resolve_secret_squote" $username }};
     {{- end }}
 {{- end }}
 
+ALTER USER 'root'@'localhost' IDENTIFIED VIA unix_socket;
```

Comment:
in addition to this, `'root'@'` with password will be added after https://github.com/sapcc/helm-charts/pull/8096